src/Opdex.Platform.WebApi/Controllers/StatusController.cs

```diff
@@ -27,7 +30,8 @@ public ActionResult<StatusResponseModel> GetStatus()
         return new StatusResponseModel
         {
             Commit = _opdexConfiguration.CommitHash,
-            Identifier = _opdexConfiguration.InstanceId
+            Identifier = _opdexConfiguration.InstanceId,
+            UnderMaintenance = _maintenanceConfig.Value.Locked
```

Comment:
Wondering if we should move, or also add this to the Index status endpoint.

Also considering preventing Index if either `Available` is false or if `Maintenance` is true. What do you think?